### PR TITLE
Add an animated version of the update icon

### PR DESCRIPTION
**Pull request by jimmynotjim**
When we released the svg icons we skipped the animated icons for the time
being. Turns out the update icon is realistically the only one that
needs to be animated. Duplicating the static icon with a modifier class
and then targeting that in the Less is the simplest solution.

## Additions

- `cf-icon-updating` SVGs
- `cf-icon-svg__updating` styles

## Testing

1. Follow the [testing locally instructions](https://github.com/cfpb/capital-framework/blob/canary/CONTRIBUTING.md#testing-components-locally)
1. Run `gulp build` && `npm start`
1. Navigate to `http://localhost:3000/components/cf-icons/#status-icons`
1. Modify the class of the existing updating icon to include `cf-icon-svg__updating` (there's a weird jekyll bug where if I added the new icons on their own row it would render each and the name as the first column of multiple rows. Will investigate a fix outside of this PR).

## Checklist

- [x] PR has an informative and human-readable title
- [x] Changes are limited to a single goal (no scope creep)
- [x] Code can be automatically merged (no conflicts)
- [x] Code follows the standards laid out in the [development playbook](https://github.com/cfpb/development)
- [x] Passes all existing automated tests
- [x] Any _change_ in functionality is tested
- [x] New functions are documented (with a description, list of inputs, and expected output)
- [x] Placeholder code is flagged / future todos are captured in comments
- [x] Visually tested in supported browsers and devices (see checklist below :point_down:)
- [x] Project documentation has been updated (including the "Unreleased" section of the CHANGELOG)
- [x] Reviewers requested with the [Reviewers tool](https://help.github.com/articles/requesting-a-pull-request-review/) :arrow_right:

### Browsers

- [x] Chrome
- [x] Firefox
- [x] Safari
- [x] Internet Explorer 8, 9, 10, and 11 (where animation is supported otherwise it's static)
- [x] Edge
- [x] iOS Safari
- [ ] Chrome for Android

### Accessibility

- [x] Keyboard friendly
- [x] Screen reader friendly

### Other

- [ ] Is useable without CSS
- [x] Is useable without JS
- [x] Flexible from small to large screens
- [x] No linting errors or warnings
- [x] JavaScript tests are passing

### CODE REVIEW

File: src/cf-icons/src/cf-icons.less [hunk 1]
```diff
@@ -38,6 +38,20 @@
...
+        transform-origin: 50% 50%;
+    }
+}
+
+@keyframes updating-animation {
+    0% {
+        transform: rotate( 0deg );
+    }
+    100% {
+        transform: rotate( 359deg );
```
Comment (wpears) [inline src/cf-icons/src/cf-icons.less]: I feel like this should be `transform: rotate( 360deg );`, though I haven't seen a case made either way. Conceptually, only going to 359 would mean that there'd be a 1 degree jump at the start of the loop (from 359 to 0), but I know there are certain animation frame hacks that require dropping a degree to make the end result seem smoother, but I dunno if that is true in this case.

**Comment by sebworks** [comment]
Is it worth using`will-change`? I know it isn't supported by all the browsers though.

File: src/cf-icons/src/cf-icons.less (hunk 1 above)
Comment (jimmynotjim) [reply src/cf-icons/src/cf-icons.less]: No, 360 and 0 are the same position, so if you do 360 there's a pause at the end rather than smooth animation.

**Comment by jimmynotjim** [comment]
> Is it worth using `will-change`? I know it isn't supported by all the browsers though.

There's no reason to use `will-change` on the icon, because the icon is always animated. The entire `svg` is replaced if the item is no longer being updated. Notification container should probably get `will-change` though, because it's typically changing icons and text.

**Pull request merged**